### PR TITLE
Client SDK guides - "Make a Call" and "Receive a Call"

### PR DESCRIPTION
**Pull request by pardel**
## Description

Refactored and updated existing make and receive calls guides for the Client SDK.

## Review

* [Make a Call](https://nexmo-developer-pr-2330.herokuapp.com/client-sdk/in-app-voice/guides/make-call)
* [Receive a Call](https://nexmo-developer-pr-2330.herokuapp.com/client-sdk/in-app-voice/guides/receive-call)

### CODE REVIEW

File: config/redirects.yml [hunk 1]
```diff
@@ -255,6 +255,6 @@
 "/en/voice/sip/concepts/sip-connect": "/en/voice/sip/concepts/programmable-sip"
 "/concepts/guides/webhooks#using-ngrok-for-local-development": "/en/concepts/guides/testing-with-ngrok"
 "/messaging/sms/message-search/api-reference": "/api/developer/messages"
-"/client-sdk/in-app-voice/guides/start-and-receive-calls/android": "/client-sdk/in-app-voice/guides/make-call/android"
```
Comment (fabianrbz) [inline config/redirects.yml]: @pardel this should work, did you get an error?

Comment (pardel) [reply config/redirects.yml]: @fabianrbz yes, I did get an error for that. It said that "/client-sdk/in-app-voice/guides/make-call/android.md" doesn't exist.

Comment (pardel) [reply config/redirects.yml]: @fabianrbz Found the logs:
```
Specified redirect could not be found: /github/workspace/_documentation/en/client-sdk/in-app-voice/guides/make-call/android.md
Specified redirect could not be found: /github/workspace/_documentation/en/client-sdk/in-app-voice/guides/make-call/swift.md
Specified redirect could not be found: /github/workspace/_documentation/en/client-sdk/in-app-voice/guides/make-call/javascript.md
##[error]Docker run failed with exit code 1
```

**Comment by fabianrbz** [comment]
I'll take a look. It might be an issue with the GitHub Action, we are
constantly improving them. Do you know if the redirect actually worked in
the app itself?

**Comment by pardel** [comment]
@fabianrbz sorry, not sure...

**Comment by fabianrbz** [comment]
Don't worry I'll check.

**Comment by fabianrbz** [comment]
@pardel the redirect works in the app, so definitely an issue with the GA. can you remove d37b67b7aa2b1b7972eba7845a675e95177d3d4e ? I'll take care of the rest

**Comment by pardel** [comment]
@fabianrbz done

**Pull request merged**